concordia/signals/handlers.py

```diff
@@ -89,6 +89,24 @@ def send_asset_reservation_released(sender, **kwargs):
         user_pk=kwargs["user_pk"],
     )
 
+    no_activity = True
+    # TODO: determine whether the user saved any transcriptions
+    # before releasing the reservation.
+    # If not, then no_activity should be true
+
+    if no_activity:
+        asset = Asset.objects.filter(pk=kwargs["asset_pk"]).annotate(
+            contributor_count=Count("transcription__user", distinct=True),
```

Comment:
@acdha Please tell me there's a better alternative than what I'm doing here. To me, this is terrible.